tests/utils/exec/cluster.go

```diff
@@ -110,11 +113,9 @@ func NewClusterService(manifestDir string) *ClusterService {
 	return sc
 }
 
-//******************************************************
-
 func CreateTFCluster(ctx context.Context, manifestsDir string,
 	varArgs map[string]interface{}, abArgs ...string) (string, error) {
-	targetDir := CON.GrantClusterManifestDir(manifestsDir)
+	targetDir := CON.GrantClusterManifestDir(CON.ROSAClassic)
```

Comment:
manifestsDir is required parameter, and we will have multiple manifests dirs. please don't hardcode it to rosaclassic